libs/api-client-tiger/src/gd-tiger-model/VisualizationObjectModelV2.ts

```diff
@@ -28,7 +28,7 @@ export namespace VisualizationObjectModelV2 {
 
     export interface IAttribute {
         localIdentifier: Identifier;
-        displayForm: ObjectIdentifier;
+        label: ObjectIdentifier;
```

Comment:
This is breaking. We should either create V3 or take the risk that the data created in the last week will turn unusable.

Reply:
This is breaking also on AFM API level (together with the other sorting change). So I think breaking it here doesn't cause additional problems. (For the context we want to do as much as these breaking changes before tiger GA.)

Reply:
Ok, so to confirm we are ok "nuking" the exisitng insights?